### ISSUE TITLE
deepcopy function references a non-existant variable

### ISSUE THREAD

**Issue by dhylands** [pfalcon/micropython-lib#26]
In copy.py:
https://github.com/pfalcon/micropython-lib/blob/01b5a1571d3c094074fb08727c792bfd63b1732e/copy/copy.py#L163
the deepcopy function references the variable dispatch_table which isn't initialized.

**Comment by pfalcon** [comment]
Yeah, CPython's "copy" module turned out to be very bloated and tangled, it doesn't work fully. I'm sure that someone needing deepcopy() will propose a working solution. I for one, all these years never had a need for it (I avoid using it myself, knowing how inefficient and bloated it is, and apparently, many other folks think the same, as I didn't hit it in the code I ported.)
